Source/BidMachineAdapterAd.swift

```diff
@@ -28,7 +28,14 @@ class BidMachineAdapterAd: NSObject {
 
     /// The completion for the ongoing show operation.
     var showCompletion: ((Result<PartnerEventDetails, Error>) -> Void)?
-    
+
+    /// Waterfall item price
+    var price: Double? {
+      let setting = self.request.partnerSettings["price"]
+      let priceNSNum = setting as? NSNumber
+      return priceNSNum?.doubleValue
```

Comment:
nit: the indentation should be 4 whitespaces instead of 2

Reply:
[fixed](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-bidmachine/commit/8088c75d3f4c18ced067ddd390810a406ff64828)